eZ/Publish/API/Repository/Tests/Values/User/Limitation/ObjectStateLimitationTest.php

```diff
@@ -144,4 +147,181 @@ public function testObjectStateLimitationForbid()
         $contentService->deleteContent($draft->contentInfo);
         /* END: Use Case */
     }
+
+    /**
+     * Tests an ObjectStateLimitation.
```

Comment:
Nitpick: I'd provide more verbose description in this case as it tests some specific forbidden condition, right?